Source/Core/buildModuleUrl.js

```diff
@@ -5,6 +5,8 @@ import Resource from "./Resource.js";
 
 /*global CESIUM_BASE_URL*/
 
+// TODO: all these should have JSDoc tags, or mark as private
```

Comment:
I think there were only one or two of these functions that the compiler actually noticed and complained about, but I figured while we're in there it would make sense to just mark everything.